README.md

```diff
@@ -99,7 +99,7 @@ after `--global`
 
 #### 1.2 Authentication
 
-To use balena CLI, you need to login to a balenaCloud account. If you don't
+To use balena CLI, you need to login into a balenaCloud account. If you don't
```

Comment:
"log into", or "login to", or ideally "log in to", but i don't think "login into" sounds right

Reply:
Yeah, I agree. My bad, I went through the changes a touch too fast. I will open a PR to patch this.